packages/reactDom/src/ReactServerDOM.mli

```diff
@@ -1,10 +1,17 @@
+module Env : sig
+  type t = DEV | PROD
+end
+
 val render_html :
+  ?env:Env.t ->
```

Comment:
I would rather be ~env: `Dev | `Prod, rather than ReactServerDOM.DEV and ReactServerDOM.PROD